### PR TITLE
Django Setup

### LINKED ISSUE #6: FEAT: Setup A New Django Application
- Setup a new [Django] application.
    - Make sure you configure a public port for the VM, like so:
      ```
      config.vm.network "forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
      ``` 
      That will allow us to view the Django application (that is being rendered in the VM), via our host machine on `127.0.0.1:8000`.
- Setup [Pipenv] to install and manage Django and other Python dependencies.

[Django]: https://www.djangoproject.com/
[Pipenv]: https://github.com/pypa/pipenv

> ATTENTION: This issue DEPENDS ON #2

### PR DESCRIPTION
**Pull request by paOmer**
Django framework setting up:

- Installed pipenv
- installed Django
- bootsrapped django app
- providing local http access to VM

DEPENDS ON https://github.com/redhat-beyond/Beyond-07-team-4/pull/1
Close #6

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant
```
Comment (amittcohen) [inline .gitignore:132]: Why should we ignore vagrant files?

Comment (paOmer) [reply .gitignore:132]: It's a local ".vagrant" file that creates automatically, which means every computer who will run vagrant will create a local copy of it.

**Review by amittcohen — APPROVED** [review]

**Comment by taljacob2** [comment]
Looks good!

**Review by taljacob2 — APPROVED** [review]
LGTM

**Comment by taljacob2** [comment]
> Looks good! @paOmer please mention in this PR's description, that it "_DEPENDS ON #1_"

You did it. Awesome! I opened an issue (#6) with more info about this feature.
Please edit this PR's description, and add the line:
```
Close #6 
```
This will end my review on this PR 👍

**Review by ShirleyFichman — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**